docs/tutorial/tutorial-5/android.rst

```diff
@@ -2,9 +2,234 @@
+covers running the app on a virtual Android device.
+
+On Android, Briefcase's ``run`` command requires that you specify which device to run on.
+If you use ``run`` without specifying a device, Briefcase will show a list of
+physical and virtual devices (if any) and offer to create one.
+
+.. tabs::
+
+  .. group-tab:: macOS
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase run android
+
+      Select device:
+
+        1) Create a new Android emulator
+
+      >
+
+  .. group-tab:: Linux
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase run android
+
+      Select device:
+
+        1) Create a new Android emulator
+
+      >
+
+  .. group-tab:: Windows
+
+    .. code-block:: doscon
+
+      (beeware-venv) C:\...>briefcase run android
+
+      Select device:
+
+        1) Create a new Android emulator
+
+      >
+
+Choose your desired device, or choose the option to create a new device. If you create a new
+device, you will have to provide a name (or accept the default of ``beePhone``).
+
+Once you choose a device, you will see two steps of Android booting followed by your app
+launching. First, the Android logo will appear.
+
+.. figure:: ../images/android/tutorial-5-booting.png
+   :alt: Android virtual device booting
+
+   Android virtual device booting
+
+In the second phase, the launcher screen will appear.
+
+.. figure:: ../images/android/tutorial-5-running.png
+   :alt: Android virtual device fully started, on the launcher screen
+
+   Android virtual device fully started, on the launcher screen
+
+Once the phone finishes booting, ``briefcase run`` will automatically begin
+to launch your app. You will first see a splash screen, then your app. If you
+see an error message in your console, you may need to follow the advice in the
+error message about finding the correct device name.
+
+Run the app on a physical device
+================================
+
+If you have an Android phone or tablet you want to run your app on, you can
+connect it to your development workstation, typically using a USB cable.
+Then you can use Briefcase's ``run`` command to target your physical device.
+
+If Briefcase can detect the device, it will appear in the ``run`` output.
+
+.. tabs::
+
+  .. group-tab:: macOS
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase run android
+
+      Select device:
+
+        1) CoolPhone
+        2) Create a new Android emulator
+
+      >
+
+  .. group-tab:: Linux
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase run android
+
+      Select device:
+
+        1) CoolPhone
+        2) Create a new Android emulator
+
+      >
+
+  .. group-tab:: Windows
+
+    .. code-block:: doscon
+
+      Select device:
+
+        1) CoolPhone
+        2) Create a new Android emulator
+
+      >
+
+In this example, you could type **1** then press return to run the app on your
+CoolPhone device. In the case that your device is detected, but the Android
```

Comment:
(a) Might be better to use a name from an actual device, rather than Coolphone, and mention what name will come up.
(b) Apparently, you can actually give your Android device a name - but that's *not* the name that appears here. Mentioning this because it came up when Katie did the tutorial, and she was surprised when "hammerhead" (or something like that) came up. AFAICT, the name that comes up is the device codename.

Reply:
+1. Pending.

I'll fix this once we merge a PR to use Android device model names, per discussion on Gitter.

Reply:
Fixed in this PR in 20ea4400ec4cabf848a00db75cae664d21d97f48